tests/functions.rs

```diff
@@ -382,3 +382,69 @@ fn join_argument_count_error() {
     .status(EXIT_FAILURE)
     .run();
 }
+
+#[test]
+fn test_path_exists_filepath_exist() {
+  let test = Test::new();
+  let testfilepathbuf = test.tempdir.path().join("testfile.txt");
+  let testfilepath = testfilepathbuf.as_path();
+  let _file = std::fs::File::create(testfilepath);
+  test
```

Comment:
You can use `Test::tree` to populate the tempdir the test runs in with files:

```suggestion
  Test::new()
    .tree(tree! {
      testfile: ""
    })
```